src/components/datepicker/utils.js

```diff
@@ -0,0 +1,15 @@
+import { format } from 'date-fns';
+
+export const currentTimeDot = () => {
+	return (
+		<span className="bg-blue-500 h-1 w-1 absolute rounded-full inline-block bottom-0 left-1/2 right-1/2"></span>
```

Comment:
❌ `bg-blue-500` Please check all the color class names and use the defined colors.

Reply:
it is same in figma, let me check